### ISSUE TITLE
Incorrect value for 'autocomplete' attribute in graphiql-explore plugin

### ISSUE THREAD

**Issue by jair1823** [OneGraph/graphiql-explorer#96]
**Description**

I ran the IBM Equal Accessibility Checker tool on my website and it identified an accessibility error in the graphiql-explore plugin. Specifically, the 'autocomplete' attribute has an incorrect value.

According to the W3C accessibility guidelines, the 'autocomplete' attribute should be set to 'off' for inputs that are not intended to be autocompleted. However, the current value in the graphiql-explore plugin is not compliant with this guideline.

I believe that this error is preventing some users from being able to interact with this feature of the website. Therefore, I would like to request that the 'autocomplete' attribute in the graphiql-explore plugin be updated to the correct value.

**Steps to Reproduce**

- Navigate to the graphiql-explore plugin
- Inspect the input fields for the 'autocomplete' attribute
- Observe that the value is incorrect

**Expected Results**

The 'autocomplete' attribute in the graphiql-explore plugin should have the correct value according to the W3C accessibility guidelines.

<img width="1167" alt="image" src="https://user-images.githubusercontent.com/39355524/227347711-381e7e89-95f6-4db1-acff-f9f73c5ac7b7.png">
